### PR TITLE
Conversation

### PR DESCRIPTION
**Pull request by JorgeMariles**
Migration

### CODE REVIEW

**Review by JorgeMariles — COMMENTED** [review]
Se creo el action server para el intent de "bring something". La funcion de action crea un archivo json que es guardado en la carpeta /jsons. El archivo json contiene el intent y las entidades

File: rasa/config.yml [hunk 1]
```diff
@@ -14,19 +14,17 @@ pipeline: null
...
 #     threshold: 0.3
 #     ambiguity_threshold: 0.1
 policies:
 - name: MemoizationPolicy
 - name: TEDPolicy
-- name: RulePolicy
-  nlu_threshold: 0.6
-  core_threshold: 0.3
-  fallback_action_name: action_default_fallback
+#- name: RulePolicy
```
Comment (JorgeMariles) [inline rasa/config.yml:27]: Estos comentarios estan mal

File: .gitignore [hunk 2]
```diff
@@ -37,6 +37,8 @@ wrokspace.code-workspace
 
 ### Others ###
 /rasa/venv/
+/rasa/models/
+/rasa/results/
```
Comment (Josecisneros001) [inline .gitignore:41]: The files are still tracked. Remove cache and do another commit.
This is how cache can be removed. 
git rm -r --cached .

**Review by AuroTB — COMMENTED** [review]
Please update description.

I'm having some trouble training, I don't know if you have the same problem or have encountered it before:
Screenshot from 2021-04-21 00-01-06
Screenshot from 2021-04-21 00-00-51

**Comment by Josecisneros001** [comment]
Wiki Entry 
Speech Tab
image
